objects/obj_control/Other_2.gml

```diff
@@ -2,10 +2,14 @@
 
 // Game (Global) variables
 global.playerControl = true;
+global.isSeqActive = false;
 
 global.gameOver = false;
 global.gameStart = false;
 
+global.CamX = undefined;
+global.CamY = undefined;
```

Comment:
experimental code

Reply:
did the experiment code work? what's it experimenting?